guides/common/modules/ref_automatic-contexts-for-discovered-hosts.adoc

```diff
@@ -1,17 +1,17 @@
 [id="Automatic_Contexts_for_Discovered_Hosts_{context}"]
 = Automatic contexts for discovered hosts
 
-{ProjectServer} assigns an organization and location to discovered hosts according to the following sequence of rules:
+{ProjectServer} assigns an organization and location to discovered hosts automatically according to the following sequence of rules:
 
 . If a discovered host uses a subnet defined in {Project}, the host uses the first organization and location associated with the subnet.
-. If the `discovery_organization` or `discovery_location` fact values are set, the discovered host uses these fact values as an organization and location.
-To set these fact values, navigate to *Administer* > *Settings* > *Discovered*, and add these facts to the *Default organization* and *Default location* fields.
-Ensure that the discovered host's subnet also belongs to the organization and location set by the fact, otherwise {Project} refuses to set it for security reasons.
-. If none of the previous conditions exists, {Project} assigns the first Organization and Location ordered by name.
+. If the default location and organization is configured in global settings, the discovered hosts are placed in this organization and location.
+To configure these settings, navigate to *Administer* > *Settings* > *Discovery*, and select values for the *Discovery organization* and *Discovery location* settings.
+Ensure that the discovered host's subnet also belongs to the selected organization and location, otherwise {Project} refuses to set it for security reasons.
```

Comment:
```suggestion
Ensure that the subnet of the discovered host also belongs to the selected organization and location, otherwise {Project} refuses to set it for security reasons.
```
Avoiding possesive as per the IBM style guide.